src/commands.rs

```diff
@@ -41,6 +42,9 @@ pub enum Commands {
     /// Lints a document or a directory containing documents.
     Lint(check::LintArgs),
 
+    /// Locks Docker images to a sha256 digest.
```

Comment:
Since there's not yet any enforcement mechanism for the lock files, we should try out best to make sure no users see the docs for a new command and get confused why it's not having an effect.

```suggestion
    /// (Experimental) Locks Docker images to a sha256 digest.
```

Reply:
lol have never touched the auto-generated profile picture github assigned my account at creation. Maybe it's time to personalize the account some 🤔